ostd/src/mm/page/meta.rs

```diff
@@ -295,9 +295,8 @@ fn alloc_meta_pages(nframes: usize) -> Vec<Paddr> {
         .get()
         .unwrap()
         .lock()
-        .alloc(nframes)
-        .unwrap()
-        * PAGE_SIZE;
+        .alloc(core::alloc::Layout::from_size_align(nframes * PAGE_SIZE, PAGE_SIZE).unwrap())
+        .unwrap_or_else(|| panic!("Failed to allocate metadata pages"));
```

Comment:
Would be the same as `.expect("Failed to allocate metadata pages")`. Didn't Clippy instructed you to do so?